test/core/base/cache.js

```diff
@@ -1,14 +1,18 @@
-describe('axe._cache', function() {
+describe('axe._cache', function () {
   'use strict';
 
-  it('should set items from the cache without error', function() {
+  beforeEach(function () {
+    axe._cache.clear();
+  });
+
```

Comment:
This is handled by a [global `afterEach` in testutils](https://github.com/dequelabs/axe-core/blob/develop/test/testutils.js#L565)

```suggestion
```